src/python/pyproject.toml

```diff
@@ -47,7 +47,7 @@ demos = [
   "flask",
   "sacremoses",
   "pyqt5",
-  "sentence-splitter@git+https://github.com/mediacloud/sentence-splitter",
+#  "sentence-splitter@git+https://github.com/mediacloud/sentence-splitter",
```

Comment:
Please explain in the comment why it's commented (because not used yet but will be in pymarian-webapp?) or remove.